README.rst

```diff
@@ -10,6 +10,21 @@ core `astropy`_ project since it is of use more generally.
 .. _pytest: https://pytest.org/en/latest/
 .. _astropy: https://astropy.org/en/latest/
 
+IMPORTANT: Retirement Roadmap
+-----------------------------
+
+As of https://github.com/astropy/astropy/pull/14041 , this package is no
+longer used in ``astropy`` core library. We strongly advise any packages
+that still use this package to migrate away from it. When ``astropy`` 6.0
+is released (tentatively Nov 2023), none of the active development branches
+of the core library would use this package anymore.
+
+After that, we will just do one last release of ``pytest-openfiles``
+and archive this repository.
+
+See https://docs.astropy.org/en/latest/development/testguide.html#testing-for-open-files
+on how to test for open files without this package.
```

Comment:
I think it would be good to have an example here in this repo that is concrete.  I.e.:


For open file testing, use Python's built in warning-to-error handling switch `-W`.  Pytest invocations support this as well:

```
pytest -W error::ResourceWarning
```

or set it up in your setup.cfg:

```
[tool:pytest]
filterwarnings =
    error::ResourceWarning
```

For more detail on `-W`, see https://docs.python.org/3/using/cmdline.html#cmdoption-W

Or something similar.

Reply:
OK done. Thanks for the suggestion!